contrib/craftscripts/SUBMITTING.md

```diff
@@ -1,5 +1,5 @@
 Write a cool script? You can submit a pull request to [our GitHub Repository](https://github.com/IntellectualSites/FastAsyncWorldEdit). 
 We will consider your script for inclusion in the FastAsyncWorldEdit repository. CraftScripts in the FastAsyncWorldEdit repository are
-licensed under GPLv3, like the rest of FastAsybcWorldEdit.
+licensed under GPLv3, like the rest of FastAsyncWorldEdit.
```

Comment:
It is just this typo so I don't think that I needed an issue before creating this pull request.